### PR TITLE
Added relational data sorting feature

### PR DESCRIPTION
**Pull request by erdembas**

### CODE REVIEW

File: src/LightQuery.Shared/QueryableProcessor.cs [hunk 1]
```diff
@@ -7,6 +9,43 @@ namespace LightQuery.Shared
 {
     public static class QueryableProcessor
     {
+        public static PropertyInfo GetPropertyInfoRecursivly(this IQueryable queryable, String propName)
```
Comment (berkayakcay) [inline src/LightQuery.Shared/QueryableProcessor.cs]: The name should be "**GetPropertyInfoRecursively**"

Comment (erdembas) [reply src/LightQuery.Shared/QueryableProcessor.cs]: Fixed it 👍

File: test/LightQuery.Shared.Tests/QueryableProcessorTests.cs [hunk 2]
```diff
@@ -13,6 +13,83 @@ private class User
...
+        private class ProductDetail
+        {
+            public string Barcode { get; set; }
+            public string ManufacturerCode { get; set; }
+        }
+
+        // Relational Data Sorting Sample Data
+        private IQueryable<Order> GetOrders()
+        {
+            return new[]
```
Comment (GeorgDangl) [inline test/LightQuery.Shared.Tests/QueryableProcessorTests.cs]: Could you append something like `.OrderBy(c => Guid.newGuid())`, just to ensure that it's not coming back in the order it's defined? The test below that checks if it's order by title would succeed even if no ordering is applied at all currently.

**Comment by GeorgDangl** [comment]
This looks very good, thank you very much! If you update the test, I'll merge it in dev and will add some integration tests to make sure it works with Entity Framework Core.

File: test/LightQuery.Shared.Tests/QueryableProcessorTests.cs (hunk 2 above)
Comment (erdembas) [reply test/LightQuery.Shared.Tests/QueryableProcessorTests.cs]: ```.OrderBy(x=> Guid.NewGuid())``` added 👍

**Comment by GeorgDangl** [comment]
Thank you very much! I'll add integration tests and will then publish to NuGet later. Dev builds should be available on the MyGet feed in a few minutes.

**Comment by GeorgDangl** [comment]
v1.5.0 is now available. I've added null-checks when relational queries are applied, to guard against `NullReferenceException`s

**Pull request merged**